src/common/runnable/run-many-for.ts

```diff
@@ -3,46 +3,81 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
```

Comment:
Blocker: checked the unit tests, and they are not sufficient to cover this key technical feature of Lens.

Reply:
Oh okay, will add more then. Thought they were